Sources/App/Views/DocumentationPageProcessor.swift

```diff
@@ -151,7 +154,7 @@ struct DocumentationPageProcessor {
                         )
                     )
```

Comment:
There's an `unwrap` of `referenceKind` in this file (because `referenceKind` used to be optional) that should be redundant now (line 117, I can't reference it directly because it's not new in the diff).